fieldsolver/gridGlue.hpp

```diff
@@ -168,7 +168,8 @@ template< unsigned int numFields > void getFieldDataFromFsGrid(
       int nCellsToSum = 0;
       
       for(int iCell = 0; iCell < nCells; ++iCell) {
-         if ((transferBufferPointerTechnical[i] + iCell)->sysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE) {
+         if ((transferBufferPointerTechnical[i] + iCell)->sysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE ||
+              (transferBufferPointerTechnical[i] + iCell)->sysBoundaryFlag == sysboundarytype::BOUNDARY_PADDING) {
```

Comment:
And this is also wrong, methinks, you *want* to have the `BOUNDARY_PADDING` cells in the averaging.

Reply:
Right, this doesn't make a lot of sense, neither in its current, nor in the previous version. But it seems this function never gets called anyway, so I suggest just chopping the whole function away.